assets/src/data/eventespresso/base-selectors.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { select } from '@wordpress/data';
+
+/**
+ * Invokes the selector for whether a given selectorName in a given registered
+ * reducer store is in the midst of resolving.
+ * @param {string} reducerKey
+ * @param {string} selectorName
+ * @param {*[]} args
+ * @return {boolean}  True means the selector is resolving. False means it
+ * either hasn't started yet or is complete.
```

Comment:
No.  There is another state that can existing for resolution of selectors "hasStarted` (which is when the resolution state is undefined in the `core/data` store for that selector.   It's not reflected in our stores because I don't care about that state.  I only care:

- Is it resolving (meaning its in the process of being retrieved via the api etc).
- Is it finished resolving (meaning it has been resolved... currently core/data never changes that state but there's a pull in progress to allow for resetting the resolution state).
 
The related `core/data` selectors can be found in `gutenberg/packages/data/src/store/selectors`  The particular one for this method has this signature:

```js
export function isResolving( state, reducerKey, selectorName, args = [] ) {
	return getIsResolving( state, reducerKey, selectorName, args ) === true;
}
```

Does that explanation help?

Reply:
I pointed to the code you could look at because I thought it might help clarify. I'm assuming you haven't looked at all the selectors in the file I referenced based on your answer.  I don't know why you consider the doc comment I added in accurate. The core `isResolving` class which this code block calls explicitly does a type-safe boolean check on whether `getIsResolving` returns `true`.  So it will only return `true` or `false`.  Resolver state can be `undefined`, `false`, or `true`.  `undefined` means it is not started yet.  `false` means it has finished resolving.  `true` means it is in the midst of resolving.  Since `isResolving` is simply checking for whether the state of resolution for our selector is `true`.  Then if that function returns `true` it means we definitely know that the selector is _being_ resolved.  If it returns false then we don't know whether it has started or is being resolved because its NOT returning the state of resolution. It's only returning whether its is being resolved or not.

So imo the doc block is accurate.

Reply:
After discussing this in slack, I'm going to change this to be the same as the comments used in the `core/data` resolution selectors.  So for this method it will be:

> Whether resolution is in progress.